deployments/common/crds/appprotectdos.f5.com_dosprotectedresources.yaml

```diff
@@ -31,7 +31,7 @@ spec:
             metadata:
               type: object
             spec:
-              description: DosProtectedResourceSpec deines the properties and values a DosProtectedResource can have.
+              description: DosProtectedResourceSpec defines the properties and values a DosProtectedResource can have.
```

Comment:
the CRD are automatically generated, so we need to change the source

pinging @rafwegv